ui/core/src/model/calculations.test.ts

```diff
@@ -12,126 +12,299 @@
 // limitations under the License.
 
 import { TimeSeriesValueTuple } from '@perses-dev/core';
-import { CalculationsMap, CalculationType } from './calculations';
+import { CalculationsMap, CalculationType, getCalculation, getCalculations } from './calculations';
 
 interface CalculationTestCase {
   values: TimeSeriesValueTuple[];
   calculation: CalculationType;
   expected: number | undefined | null;
 }
 
```

Comment:
The diff looks a little weird because I moved the test cases around. A bunch of these existed already and I added a bunch of tests cases for edge cases (e.g. all nulls, empty array) and new calculations (min, max).